public/content/roadmap/dencun/index.md

```diff
@@ -0,0 +1,121 @@
+---
+title: Cancun-Deneb (Dencun) FAQ
+description: Frequently asked questions regarding the Cancun-Deneb (Dencun) network upgrade
+lang: en
+---
+
+# Cancun-Deneb (Dencun) {#dencun}
+
+Cancun-Deneb (Dencun) is an upgrade to the Ethereum network which activates **Proto-Danksharding (EIP-4844)**, introducing data **blobs** for cheaper layer 2 (L2) rollup storage.
+
+A new transaction type enables rollup providers to store data in a more cost-effective manner, in what are known as "blobs", which are guaranteed to be made available to the network, but can be discarded after a couple weeks.
+
+This significantly reduces the cost of using rollups, limits chain growth, and helps to support more users while maintaining security and a decentralized set of node operators.
+
+## When do we expect that rollups will reflect lower fees as a result of Proto-Danksharding? {#when}
+
+- This upgrade is set for activation at start of [epoch](/glossary/#epoch) 269568, occurring on **13-Mar-2024 (UTC)**
```

Comment:
maybe date (13-Mar-2024) should be formatted according to locale

Reply:
Yeah, agree... this isn't really possible without some JavaScript so I put up #12426 to add a custom component that maps to the `time` html element to dynamically render a markdown date according to `locale`